core/src/main/scala/org/genivi/sota/core/rvi/TransferController.scala

```diff
@@ -180,7 +178,9 @@ class TransferProtocolActor(db: Database, rviClient: RviClient,
             db.run(OperationResults.persist(org.genivi.sota.core.data.OperationResult(
```

Comment:
@rdanitz could you check if this code is equivalent? https://gist.github.com/simao/7f73f246b6c75eec5f5c